PageObjects/essui-encselectionpageObjects.json

```diff
@@ -0,0 +1,20 @@
+{
+    "encselectionpageObjectsConfig": 
+    {        
+        "encNameSelector":"text=ENC name",
+        "startLinkSelector":".linkStartAgain",
+        "textAboveTableSelector":"text=Select up to 100 ENCs and make an exchange set",
+        "textAboveTable":"Select up to 100 ENCs and make an exchange set",  
+        "rightTableRowsCountSelector":"//div/div[2]/div[3]/div[1]/table/tbody/tr",
+        "rightTableEncPositionSeletor":"//div/div[2]/div[3]/div[1]/table/tbody/tr[1]/td[1]",
+        "firstCheckBoxSelector":"[aria-label='AU220150'] input[type='checkbox']",
```

Comment:
can we please change this locator to generic. This currently has enc number. should we change the test data in future, the test will fail as locator is not valid.

Reply:
Hello Ravi,
Added generic locator. please Review.